Copyedit on the Philadelphia story

randomecho opened the pull request:
- Officer as position, not Office
- "As if this all wasn't awesome enough" was a little hard to parse. Hoping this version catches the awesome in the same voice.
- Missing "of" in City of Philadelphia

afeld commented on _posts/2013-09-26-philadelphia-gets-going-and-gets-open.md:
```diff
@@ -7,16 +7,16 @@ image: /images/philadelphia.png
+Philadelphia is also using GitHub to store and host the data they're opening up. Headd says, "the amazing support for GeoJSON that was announced earlier this year did it for us. We started posting some of our [geospatial data](https://github.com/CityOfPhiladelphia/phl-open-geodata) to GitHub, and we're encouraging our data community to help us convert some of the Shapefiles we already make public ...
 
-As if this all wasn't awesome enough, Headd and his team have posted their [Open Government Plan](https://github.com/CityOfPhiladelphia/open-gov-phl) and are using GitHub Issues to get feedback and data requests _and_ are using GitHub to experiment with better procurement. You can read his overview of the process [here](http://civic.io/2013/03/27/experiments-in-github-based-procurement/).
+The awesome doesn't stop there as Headd and his team have posted their [Open Government Plan](https://github.com/CityOfPhiladelphia/open-gov-phl) and are using GitHub Issues to get feedback and data requests _and_ are using GitHub to experiment with better procurement. You can read his overview of the process [here](http://civic.io/2013/03/27/experiments-in-github-based-procurement/).
```
I think I liked this line better before.  "The awesome" doesn't quite fit the voice of the rest of the post.

jlord commented:
Thanks @randomecho! I'm :thumbsup: with the changes but agree with @afeld re: the awesome line. Maybe change the orig to "As if all _of_ this _work_ wasn't awesome enough," (emphasis = additions)

randomecho committed:
Copyedit on Philadelphia story

- Officer as position, not Office
- "As if this all wasn't awesome enough" was a little
  hard to parse. Updated to suggested new wording by @jlord
- Missing "of" in City of Philadelphia

randomecho commented:
Good call @jlord and @afeld and have leaned in with the suggested update to the updated awesome line.

jlord commented:
Thanks so much @randomecho!!
